patches/server/0055-Add-exception-reporting-event.patch

```diff
@@ -167,14 +167,15 @@ index ed6203119fe2e4ee47a2d51c84df5b7c236f32da..108c7ec010f826f9b9bb2db0987d2bcc
 +                "Attempted to place a tile entity (" + blockEntity + ") at " + blockEntity.getBlockPos().getX() + ","
 +                    + blockEntity.getBlockPos().getY() + "," + blockEntity.getBlockPos().getZ()
 +                    + " (" + getBlockState(blockposition) + ") where there was no entity tile!\n" +
-+                    "Chunk coordinates: " + (this.chunkPos.x * 16) + "," + (this.chunkPos.z * 16));
++                    "Chunk coordinates: " + (this.chunkPos.x * 16) + "," + (this.chunkPos.z * 16) +
++                    "\nWorld: " + this.level.getWorld().getName());
```

Comment:
Should probably use the key and not the name

Reply:
Might be `level.getWorld().getKey()`...

Reply:
Do `this.level.dimension().location()`

Reply:
Done